### PR TITLE
Update normalize to 6.0.0

### PR DESCRIPTION
**Pull request by simonsmith**
Also adds the reset rules that were removed from normalize

See https://github.com/necolas/normalize.css/issues/664

@giuseppeg

### CODE REVIEW

File: lib/base.css [hunk 1]
```diff
@@ -80,3 +82,37 @@ ul {
...
+/**
+ * Change the font styles in all browsers.
+ */
+
+button,
+input,
+optgroup,
+select,
+textarea {
+  font-family: sans-serif;
```
Comment (giuseppeg) [inline lib/base.css]: should this be `inherit`?

File: lib/base.css [hunk 2]
```diff
@@ -80,3 +82,37 @@ ul {
...
+}
+
+/**
+ * Change the border, margin, and padding in all browsers.
+ */
+
+fieldset {
+  border: 1px solid #c0c0c0;
+  margin: 0 2px;
+  padding: 0.35em 0.625em 0.75em;
```
Comment (giuseppeg) [inline lib/base.css]: magic numbers (same for the 2px margin)?

File: lib/base.css (hunk 1 above)
Comment (simonsmith) [inline lib/base.css]: This is just copied straight from normalize. They were the opinionated rules removed in `6.0.0`. There is a bit of contention over whether that should have happened so in the meantime I think they can live in `base`.

File: lib/base.css (hunk 2 above)
Comment (simonsmith) [inline lib/base.css]: Same as above

File: lib/base.css (hunk 1 above)
Comment (giuseppeg) [inline lib/base.css]: got it, let's merge then :)

**Pull request merged**